.github/workflows/ignore-phpunit.yml

```
@@ -0,0 +1,18 @@
+name: PHPUnit
+
+on:
+  pull_request:
+    branches:
+      - develop
+    paths-ignore:
+      - '**.php'
+
+jobs:
+  phpunit:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: ['7.4', '8.0']
+    steps:
+      - run: 'echo "No PHPUnit check required"'
```

```suggestion
      - run: 'echo "No PHPUnit check required"'

```

Nit: EOF line break would be lovely